apps/api/src/app/events/e2e/trigger-event.e2e.ts

```diff
@@ -779,14 +781,14 @@ describe('Trigger event - /v1/events/trigger (POST)', function () {
       channel: channelType,
     });
 
-    const block = message.content[0] as IEmailBlock;
+    const block = message?.content[0] as IEmailBlock;
 
     expect(block.content).to.equal('Hello John Doe, Welcome to Umbrella Corp');
   });
 
   it('should throw an error when workflow identifier provided is not in the database', async () => {
     const response = await session.testAgent
-      .post(`/v1/events/trigger`)
+      .post(eventTriggerPath)
```

Comment:
One of the things I like about you is that you always leave the place cleaner than when you got to it :)